plugins/org.yakindu.sct.doc.user/src/user-guide/sctunit.textile

```diff
@@ -5,370 +5,941 @@ p.
 
 ==<!-- End sctunit_keyword_raise -->==
+
+
+
+==<!-- Start sctunit_keyword_proceed -->==
+
+h4(#sctunit_proceeding_a_state_machine). Proceeding a state machine
+
+The tested state machine does not execute any run-to-completion steps (RTC) by itself, but only if being told so by way of the _proceed_ statement. The latter can advance the state machine by a specified number of RTCs or by a certain period of time.
+
+==<div class="example">==
+
+Example:
+
+bc. raise operate
+raise user.click
+proceed 1 cycle
+
+This example raises the _operate_ and _user.click_ events. The subsequent statement @proceed 1 cycle@ causes the state machine to perform one run-to-completion step, potentially acting on the raised events.
+
+==</div>==
+
+The _proceed_ statement consists of the keyword @proceed@ and an indication by what to proceed. Two variants are available:
+* @proceed@ _number_ @cycle@ – This variant instructs the state machine to perform _number_ run-to-completion steps.
+* @proceed@ _number_ _time_unit_ – This variant instructs the state machine to proceed by the specified time, e.g. @proceed 30 s@ proceeds by 30 seconds. Supported time units are:
+** @s@ – seconds
+** @ms@ – milliseconds
+** @us@ – microseconds
+** @ns@ – nanoseconds
+
+When running an SCTUnit test, the state machine's concept of "time" is decoupled from real time. That is, a statement like @proceed 3600 s@ does not have to wait for one hour of real time to elapse. Instead the state machine "leaps" by one hour in an instant, raises all affected time events, and processes them.
+
+Figure "&quot;Proceed statement grammar&quot;":#sctunit_fig_proceed_statement_grammar summarizes the structure of the _proceed_ statement:
+
+p(#sctunit_fig_proceed_statement_grammar). 
+
+!images/sctunit_grammar_030_proceed.png(Proceed statement grammar)!
+
+p=. Proceed statement grammar
+
+==<!-- End sctunit_keyword_proceed -->==
+
+
+
+==<!-- Start sctunit_keyword_var -->==
+
+h4(#sctunit_defining_variables_and_constants). Defining variables and constants
+
+Variables and constants can be defined as specified in the statechart language, please see sections "&quot;Variables&quot;":../user-guide/statechart_language.html#variables and "&quot;Constants&quot;":../user-guide/statechart_language.html#constants for all the details.
+
+However, variables in the SCTUnit language _must_ be initialized. For example, while a definition like
+
+bc. var sum: integer
+
+is fine in the statechart language, it is an error in the SCTUnit language. You would rather have to write something like
+
+bc. var sum: integer = 0
+
+Figure "&quot;Variable definition statement grammar&quot;":#sctunit_fig_variable_definition_statement_grammar summarizes the structure of the _variable definition_ statement:
+
+p(#sctunit_fig_variable_definition_statement_grammar). 
+
+!images/sctunit_grammar_020_variabledefinitionstatement.png(Variable definition statement grammar)!
+
+p=. Variable definition statement grammar
+
```

Comment:
Maybe reference to the documentation of the Stext language (if one exists) and to VariableDefinition?